wildfly-mixin/dashboards/wildfly-overview.libsonnet

```diff
@@ -0,0 +1,709 @@
+        fillOpacity: 0,
+        gradientMode: 'none',
+        hideFrom: {
+          legend: false,
+          tooltip: false,
+          viz: false,
+        },
+        lineInterpolation: 'linear',
+        lineWidth: 1,
+        pointSize: 5,
+        scaleDistribution: {
+          type: 'linear',
+        },
+        showPoints: 'auto',
+        spanNulls: false,
+        stacking: {
+          group: 'A',
+          mode: 'none',
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      decimals: 0,
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+          {
+            color: 'red',
+            value: 80,
+          },
+        ],
+      },
+    },
+    overrides: [],
+  },
+  options: {
+    legend: {
+      calcs: [],
+      displayMode: 'list',
+      placement: 'bottom',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'single',
+      sort: 'none',
+    },
+  },
+};
+
+local rejectedSessionsPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'increase(wildfly_undertow_rejected_sessions_total{deployment=~"$deployment", job=~"$job", instance=~"$instance"}[$__rate_interval])',
+      datasource=promDatasource,
+      legendFormat='{{deployment}}',
+    ),
+  ],
+  type: 'timeseries',
+  title: 'Rejected sessions',
+  description: 'Number of sessions that have been rejected from a deployment over time',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'palette-classic',
+      },
+      custom: {
+        axisCenteredZero: false,
+        axisColorMode: 'text',
+        axisLabel: '',
+        axisPlacement: 'auto',
+        barAlignment: 0,
+        drawStyle: 'line',
+        fillOpacity: 0,
+        gradientMode: 'none',
+        hideFrom: {
+          legend: false,
+          tooltip: false,
+          viz: false,
+        },
+        lineInterpolation: 'linear',
+        lineWidth: 1,
+        pointSize: 5,
+        scaleDistribution: {
+          type: 'linear',
+        },
+        showPoints: 'auto',
+        spanNulls: false,
+        stacking: {
+          group: 'A',
+          mode: 'none',
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      decimals: 0,
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+          {
+            color: 'red',
+            value: 80,
+          },
+        ],
+      },
+    },
+    overrides: [],
+  },
+  options: {
+    legend: {
+      calcs: [],
+      displayMode: 'list',
+      placement: 'bottom',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'single',
+      sort: 'none',
+    },
+  },
+};
+
+{
+  grafanaDashboards+:: {
+    'wildfly-overview.json':
+      dashboard.new(
```

Comment:
Also, to comment on the lack of logs on the shared instance, it could be due to the fact that the `filename` filter for the logs does not quite align with the prometheus scraper, since it is looking at the kubernetes wildfly pod instead of the server log directory inside that pod to scrape for logs. This should be different when running a Wildfly application itself. However, also removing the `filename` filter in the mean time should show the logs. 

The original intention in adding the `filename` filter in the first place was to filter the logs panel further and to exclude any other files that may be monitored with a simple `job` and `instance` filter.

<img width="959" alt="image" src="https://user-images.githubusercontent.com/48131175/222780178-53e99baa-d2ca-4846-92f6-8902656cd45f.png">